.travis.yml

```
@@ -1,19 +1,19 @@
-group: edge
+jobs:
+  include:
+    - stage: Tests
+      name: 'Kuzzle GraphQL Plugin Tests'
```

You don't need singlequotes here
```suggestion
      name: Kuzzle GraphQL Plugin Tests
```